### ISSUE TITLE
Image quality issues with full screen image using Deep Zoom

### ISSUE THREAD

**Issue by afollestad** [koush/ion#466]
Here's a bug report from a user that helps explain the issue.

https://plus.google.com/+Vuka%C5%A1inAn%C4%91elkovi%C4%87zavukodlak/posts/2yfKyyfFQCR

**Comment by koush** [comment]
Can you provide a sample app?

**Comment by afollestad** [comment]
Of course, this links to Google Drive: http://bit.ly/impression-0719-alpha

**Comment by koush** [comment]
I mean a standalone simple sample app with source. Or maybe just give me a test image that I can use against the ion sample?

**Comment by afollestad** [comment]
Hmm, unfortunately I don't have a simple app built. It appears people generally notice the issue with screenshots. Basically the app shows full screen images inside of Fragments that are in a ViewPager. 

It uses deep zoom and loads them into a PhotoView (https://github.com/chrisbanes/PhotoView). Wouldn't expect PhotoView to be related to quality issues.

**Comment by afollestad** [comment]
Although the users that have mentioned this issue also say that the quality issue is there initially, and goes away when they zoom in further. So it could be a deep zoom issue, or possibly a PhotoView issue.

**Comment by mightyfrog** [comment]
This is easy to reproduce. You can reuse the test code in issue #485. Just change adjustViewBounds to false in list_view_item.xml and enable/disable deepzoom. I personally accepted this behavior as a trade off though.

**Comment by eugenkiss** [comment]
I've received a report from a user with a similar problem and a test image here: https://github.com/eugenkiss/chanobol/issues/181

**Comment by Dortamur** [comment]
I am seeing a similar problem with DeepZoom enabled - when zoomed out, I'm getting a lot of aliasing artefacts, but as I zoom in closer to the native image resolution, the aliasing is reduced and then vanishes. This is more noticeable on line art, than photos.

I suspect that it's related to how the lower res views select what pixels to use. ie; just grabbing the nth pixel, vs combining a pixel cluster, and considering adjacent pixels to reduce aliasing.

It's frustrating, because my app grabs a range of images from online, which can include lower res images (which manage to look worse when DeepZoom is on), to super-high-res, as well as images that just have crazy aspect ratios (1:20) where DeepZoom is really handy to zoom in and scroll along the image.
